packages/wdio-cucumber-framework/README.md

```diff
@@ -44,70 +44,39 @@ Type: `String[]`<br />
 Default: `[]`<br />
 Example: `['@babel/register']` or `[['@babel/register', { rootMode: 'upward', ignore: ['node_modules'] }]]`
 
-### failAmbiguousDefinitions
-**Please note that this is a @wdio/cucumber-framework specific option and not recognized by cucumber-js itself**
-Treat ambiguous definitions as errors.
-
-Type: `Boolean`<br />
-Default: `false`
-
 ### failFast
 Abort the run on first failure.
 
 Type: `Boolean`<br />
 Default: `false`
 
-### ignoreUndefinedDefinitions
-**Please note that this is a @wdio/cucumber-framework specific option and not recognized by cucumber-js itself**
-Treat undefined definitions as warnings.
-
-Type: `Boolean`<br />
-Default: `false`
-
 ### name
 Only execute the scenarios with name matching the expression (repeatable).
 
 Type: `REGEXP[]`<br />
 Default: `[]`
 
-### profile
-Specify the profile to use.
-
-Type: `String[]`<br />
-Default: `[]`
-
 ### require
 Require files containing your step definitions before executing features. You can also specify a glob to your step definitions.
 
 Type: `String[]`<br />
 Default: `[]`<br />
 Example: `[path.join(__dirname, 'step-definitions', 'my-steps.js')]`
 
-### snippetSyntax
-Specify a custom snippet syntax.
-
-Type: `String`<br />
-Default: `undefined`
-
-### snippets
-Hide step definition snippets for pending steps.
+### import
+Paths to where your support code is, for ESM.
 
-Type: `Boolean`<br />
-Default: `true`
-
-### source
-Hide source uris.
-
-Type: `Boolean`<br />
-Default: `true`
+Type: `String[]`<br />
+Default: `[]`<br />
+Example: `[path.join(__dirname, 'step-definitions', 'my-steps.js')]`
 
 ### strict
 Fail if there are any undefined or pending steps
 
 Type: `Boolean`<br />
 Default: `false`
 
-### tagExpression
+### tags
```

Comment:
Can we make this backwards compatible? Possibly add a deprecation warning?

Reply:
Sure thing, I'll make sure to add a deprecation warning to give users a heads-up about the changes and ensure backwards compatibility.